test/performance/odp_l2fwd.c

```diff
@@ -1571,6 +1577,8 @@ static void parse_args(int argc, char *argv[], appl_args_t *appl_args)
 	appl_args->chksum = 0; /* don't use checksum offload by default */
 	appl_args->pool_per_if = 0;
 	appl_args->num_pkt = 0;
+	appl_args->packet_len = POOL_PKT_LEN;
+	appl_args->seg_len = POOL_PKT_LEN;
```

Comment:
This makes segment length and packet length default to the same value, which is different from segment length defaulting to packet length. To elaborate, I meant that once we've figured out what packet length is used, by processing command line parameters and checking capabilities, segment length should then default to that resulting packet length.

Reply:
Hi JereLeppanen, Thanks for detailed explanation, will integrate this logic in next version